.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 🐛 Bug Report
+about: Report an issue to help us improve
+title: ''
+labels: 'kind/bug'
+assignees: ''
+---
+**Description**
+<!-- A brief description of the issue. -->
+
+**Expected Behavior**
+<!-- A brief description of what you expected to happen. -->
+
+**Screenshots**
+<!-- Add screenshots, if applicable, to help explain your problem. -->
+
+**Enviroment:**
+- Host OS: 
+- Meshery Version: 
+- Kubernetes Version: 
+- Browser: 
+
+---
+
+[Optional] **To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+[Optional] **Additional Context**
+<!-- Add any other context about the problem here. -->
+
+---
+**Contributor Resources**
```

Comment:
@amino19 We don't need this specific section except the line below,

- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

Reply:
Good catch!